src/figuresClasses.ts

```diff
@@ -1,19 +1,90 @@
-export interface Figure {
+const rounder = (num: number): number => {
+  return Math.floor(num * 100) / 100;
+};
 
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
 }
 
-export class Triangle {
+type TColor = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: string;
+  getArea(): number;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  shape: Shape = Shape.Triangle;
+
+  constructor(
+    public color: TColor,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0 || a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error('Invalid triangle parameters');
+    }
 
+    this.color = color;
+    this.a = a;
+    this.b = b;
+    this.c = c;
+  }
+
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
```

Comment:
it is bad practice to name a variable with only one letter
so please add more descriptive name for `s`